src/Elastic.Apm.AspNetCore/ApplicationConfigurationReader.cs

```diff
@@ -4,15 +4,16 @@
 using Elastic.Apm.Logging;
 using Microsoft.Extensions.Configuration;
 
-namespace Elastic.Apm.AspNetCore.Config
```

Comment:
`Elastic.Apm.AspNetCore` and `Elastic.Apm.AspNetFullFramework` are corresponding projects, one for .NET Core, one for Full Framework. Therefore, any corresponding classes should follow a similar pattern.

- `MicrosoftExtensionsConfig` extends `AbstractConfigurationReader` and realises `IConfigurationReader`.
- `FullFrameworkConfigReader` extends `EnvironmentConfigurationReader`.

These are therefore corresponding classes between the two projects. It made sense to put both in the same `Config` subfolder in their respective projects, or to put both at root level. Since there is only one class in each project, and the class names already state they are config-related, it made sense to follow the same pattern of `FullFrameworkConfigReader` and put `MicrosoftExtensionsConfig` at root level. If you want both in a `Config` folder, then that's doable, but then `FullFrameworkConfigReader` would have to be moved to that namespace instead in order to keep alignment.

**Advantage:** It's less confusing for the reader, and they can see the correspondence between the two projects.